src/main/java/io/jenkins/plugins/conventionalcommits/ConventionalCommits.java

```diff
@@ -27,13 +27,18 @@ public Version nextVersion(Version in, List<String> commits) {
     }
 
     private boolean breakingChangeFooter(String commit){
-        int startIndex = commit.lastIndexOf("\n");
 
-        String result = commit;
+        boolean result = false;
+        String[] lines = commit.split("[\\r\\n]+");
```

Comment:
Will this work for files with different line endings?  This looks really Windows-specific.  You might need to have your regex check for linux & windows line endings.

Reply:
I believe it will work for both Linux and windows line endings...
As I've used the character class, it will work for `\r`, `\n` and `\r\n` all 3 cases :)

Re-tested it on an online tool as well ✌️ 
![image](https://user-images.githubusercontent.com/39651310/123819092-2305f400-d917-11eb-9ac0-08451034fcbd.png)